admin/admin.php

```diff
@@ -117,12 +117,19 @@ public function admin_notices() {
 	 */
 	public function add_admin_scripts( $hook ) {
 		$screen = get_current_screen();
-		if ( isset( $screen->id ) && $screen->id === 'settings_page_isc-settings' ) {
-			wp_enqueue_script( 'isc_settings_script', plugins_url( '/assets/js/settings.js', __FILE__ ), array(), ISCVERSION, true );
-		}
 
-		if ( isset( $screen->id ) && $screen->id === 'media_page_isc-sources' ) {
-			wp_enqueue_script( 'isc_sources_script', plugins_url( '/assets/js/sources.js', __FILE__ ), array(), ISCVERSION, true );
+		if ( isset( $screen->id ) ) {
```

Comment:
you could move the check to the top and return if it is false, so no brackets are needed.

Reply:
@cnhk ping